packages/common/src/testUtil/runTestSubset.ts

```diff
@@ -1,10 +1,26 @@
+import * as fs from "fs";
+import * as path from "path";
+import { getCursorlessRepoRoot } from "./getCursorlessRepoRoot";
+
 /**
- * The grep string to pass to Mocha when running a subset of tests. This grep
- * string will be used with the "Run Single Extension Test" launch
- * configuration.
- * See https://mochajs.org/#-grep-regexp-g-regexp for supported syntax
+ * Returns the grep string to pass to Mocha when running a subset of tests.
+ * @returns the grep string to pass to Mocha
  */
-export const TEST_SUBSET_GREP_STRING = "snippets";
+export function testSubsetGrepString(): string {
+  const inFile = path.join(
+    getCursorlessRepoRoot(),
+    "packages",
+    "common",
+    "src",
+    "testUtil",
+    "testSubsetGrep.txt",
+  );
+  const raw = fs.readFileSync(inFile, "utf-8");
+  const withoutComments = raw.replace(/#.*/g, "");
```

Comment:
Wouldn't all of this been simpler to do with split.map.filter instead of all the regexes?

Reply:
will change. I'm new to typescript, and poking around the internet suggested that  regular expressions were de rigeur. I'd be delighted to switch to a functional style.

Reply:
Absolutely nothing wrong with regular expressions per se, but I think we do prefer to use them only when necessary. This kind of patron is very common in javascript and something we use a lot in Cursorless.
```
raw
    .split(/\r?\n/)
    .map(line => line.trim())
    .filter(line => line.length > 0 && !line.startsWith("#"))
    .join("|")
```